bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -190,6 +200,126 @@ contract("Bridge NFT", async function(accounts) {
           );
         });
       });
+
+      describe("Fixed Fee", async function() {
+        it("Federator should receive fee", async function() {
+          const fixedFee = new BN("15");
+          let receipt = await this.bridgeNft.setFixedFee(fixedFee, {
+            from: bridgeManager,
+          });
+          utils.checkRcpt(receipt);
+
+          const tokenId = 9;

```

Comment:
maybe the `defaultTokenId` I talked about earlier would be useful now

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/248/commits/f3a764baea46195d740e416c29e6bd5fcddb5710